src/pages/Dashboard/Actions/ActionCard.vue

```diff
@@ -0,0 +1,201 @@
+        })
+      } finally {
+        this.deletingHook = false
+      }
+    },
+    editHook() {
+      this.$emit('edit-action', {
+        hook: this.hook,
+        flowConfig: this.flowConfig,
+        flowName: this.flowName
+      })
+    }
+  },
+  apollo: {
+    flowName: {
+      query: require('@/graphql/Actions/flows.gql'),
+      variables() {
+        return {
+          flowGroupId: this.hook?.event_tags?.flow_group_id
+            ? this.hook?.event_tags?.flow_group_id[0]
+            : this.flowConfig?.flow_groups[0]?.flow_group_id
+        }
+      },
+      skip() {
+        const skippy =
+          (!this.hook?.event_tags?.flow_group_id ||
+            !this.hook?.event_tags?.flow_group_id[0]) &&
+          !this.flowConfig?.flow_groups[0]?.flow_group_id
+        return skippy
+      },
+      update: data => {
+        return data.flow
+      }
+    },
+    flowConfig: {
+      query: require('@/graphql/Actions/flowConfigs.gql'),
+      variables() {
+        return {
+          flowSLAConfigId: this.hook?.event_tags?.flow_sla_config_id
+            ? this.hook?.event_tags?.flow_sla_config_id[0]
+            : ''
+        }
+      },
+      skip() {
+        return (
+          !this.hook?.event_tags?.flow_sla_config_id ||
+          !this.hook?.event_tags?.flow_sla_config_id[0]
+        )
+      },
+      update: data => {
+        return data.flow_sla_config_by_pk
+      }
+    }
+  }
+}
+</script>
+
+<template>
+  <v-card class="mb-2 headline " outlined>
+    <div class="text-right">
+      <!-- <v-switch v-model="hookSwitch" inset color="primary"></v-switch> -->
+      <v-menu :close-on-content-click="false">
+        <template #activator="{ on, attrs }">
+          <v-btn class="pa-8" icon title="More Actions" v-bind="attrs" v-on="on"
+            ><v-icon>more_horiz</v-icon></v-btn
+          ></template
+        >
+        <v-card
+          ><div
+            ><v-btn
+              :style="{ 'text-transform': 'none', 'min-width': '0px' }"
+              text
+              :loading="deletingHook"
+              color="secondaryGray"
+              @click="deleteHook"
+              ><v-icon class="pr-4">delete</v-icon>
+
+              Delete Action</v-btn
+            ></div
+          >
+          <div>
+            <v-btn
+              :style="{ 'text-transform': 'none', 'min-width': '0px' }"
+              text
+              color="secondaryGray"
+              @click="editHook"
+              ><v-icon class="pr-4">edit</v-icon>
+
+              Edit Action</v-btn
+            ></div
+          >
+        </v-card></v-menu
+      ></div
+    >
+    <div class="pl-8 pb-8 pt-0"
+      ><v-icon color="codePink" class="pr-2">{{
+        hookDetails[hook.event_type].icon
+      }}</v-icon
+      >When {{ hookType }}<span class="font-weight-bold">{{ hookName }}</span>
+      {{ hookDetail }}
+      <span v-if="includeSeconds">
+        for <span class="font-weight-bold">{{ seconds }} seconds</span> </span
```

Comment:
"starts but does not finish after 60 seconds" makes more sense to me
perhaps
"starts but does not finish in 60 seconds" ?
```suggestion
        after <span class="font-weight-bold">{{ seconds }} seconds</span> </span
```